js/index.js

```diff
@@ -1,7 +1,84 @@
 // Iteration 1: Names and Input
+const hacker1 = "Eliher";
+console.log("The driver's name is " + hacker1);
 
+const hacker2 = "Luis";
+console.log("The navigator's name is " + hacker2);
 
 // Iteration 2: Conditionals
+if(hacker1.length > hacker2.length){
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`)
+  }else if(hacker2.length > hacker1.length){
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+  }else{
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+}
 
+// Iteration 3: Loops
+let driver = hacker1;
+let name = "";
+for(let i = 0; i < driver.length; i++){
```

Comment:
esta esta muy bien pero al darle consol log se le agrega un espacio despues d la ultima letra, para solucionarlo necesitas agregarle otra condicion. Y el uppercase debe ir dentro de la funcion.
 
`for(let i = 0; i < driver.length; i++){`
  ` if(i < hacker1.length-1){`
   ` name = name + driver[i].toUpperCase() + " ";`
   `} else {`
    `name = name + driver[i].toUpperCase()`
   `}`
`}`